### DISCUSSION TITLE
is there a way to scale down the default character collider?

### DISCUSSION THREAD

**Discussion by ckdlskgkdrhd** [naverz/zepeto-studio-global#62] [General]
![ddddddddd](https://user-images.githubusercontent.com/110361648/186123811-5865a2ad-45a4-4cf7-a3de-be3bf441a80e.PNG)
the photo describes everything.

and that's with the most slimmest box collider I could make on that wall the character's standing on.

**Comment by naverz-mkao** [comment]
Hi,

Yes, you can scale the character collider! You'll notice in the Character Controller component, there's 3 public fields (Center, Radius, Height)

<img width="779" alt="image" src="https://user-images.githubusercontent.com/107931355/186156603-320bd9ad-b9c8-41eb-8399-d5e6c2d0e077.png">

You can access these fields like so

```    Start() { 
       ZepetoPlayers.instance.OnAddedLocalPlayer.AddListener(() => {
            let center:UnityEngine.Vector3 = new UnityEngine.Vector3(1,1,1);
            ZepetoPlayers.instance.LocalPlayer.zepetoPlayer.character.characterController.center = center;
            ZepetoPlayers.instance.LocalPlayer.zepetoPlayer.character.characterController.height = 2;
            ZepetoPlayers.instance.LocalPlayer.zepetoPlayer.character.characterController.radius = 0.5;
            })
       }

**Reply by ckdlskgkdrhd** [reply]
thanks again
